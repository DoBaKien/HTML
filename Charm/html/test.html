<!DOCTYPE html>
<html lang="en">

<head>
    <title>Bootstrap Example</title>
    <meta charset="utf-8">
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/css/bootstrap.min.css" rel="stylesheet">
    <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.7.1/jquery.min.js"></script>

    <link rel="stylesheet" href="https://npmcdn.com/flickity@2/dist/flickity.css">
    <link rel="stylesheet" href="../style/style.css">
</head>

<body>
    <div class="container-fuild">
        <div class="row">
            <nav class="col-sm-3" id="myScrollspy">
                <ul id="simple-list-example" class="simple-list-example-scrollspy text-center">
                    <li>
                        <a class="rounded" href="#simple-list-item-1">1 </a>
                        <a class="subtitle" href="">asasdasdadsdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-2">2</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-3">3</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-4">4</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-5">5</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-6">6</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-7">7</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                    <li>
                        <a class="rounded" href="#simple-list-item-8">8</a>
                        <a class="subtitle" href="">asasdsaddad</a>
                    </li>
                </ul>
            </nav>
            <div class="col-sm-12" style="padding: 0;">
                <div data-bs-spy="scroll" data-bs-target="#simple-list-example" data-bs-offset="0"
                    data-bs-smooth-scroll="true" class="scrollspy-example" tabindex="0">
                    <div class="box-client-1 client" id="simple-list-item-1">
                        <div class="carousel"
                            data-flickity='{ "fade": true, "imagesLoaded": true,"prevNextButtons": false, "pageDots": false ,"wrapAround": true,"adaptiveHeight": true}'>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image"
                                    src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-11.jpg" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image"
                                    src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-13.jpg" />
                            </div>
                            <div class="carousel-cell">
                                <img class="carousel-cell-image"
                                    src="../assets/imgs/hinh-nen-may-tinh-dep-full-hd-14.jpg" />
                            </div>
                        </div>
                    </div>
                    <div class="box-client-2 client" id="simple-list-item-2">

                    </div>
                    <div class="box-client-3 client">
                        <h1 id="simple-list-item-3">3</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-4 client">
                        <h1 id="simple-list-item-4">4</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-5 client">
                        <h1 id="simple-list-item-5">5</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-6 client">
                        <h1 id="simple-list-item-6">6</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-7 client">
                        <h1 id="simple-list-item-7">7</h1>
                        <p>...</p>
                    </div>
                    <div class="box-client-8 client">
                        <h1 id="simple-list-item-8">8</h1>
                        <p>...</p>
                    </div>
                </div>
            </div>
        </div>
    </div>

    <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.3.3/dist/js/bootstrap.bundle.min.js"></script>
    <script src="https://npmcdn.com/flickity@2/dist/flickity.pkgd.js"></script>
    <script src="https://unpkg.com/flickity-fade@1/flickity-fade.js"></script>
    <script>

    </script>
</body>

</html>