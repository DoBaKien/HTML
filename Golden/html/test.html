<!DOCTYPE html>
<html lang="en">

<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Document</title>
    <link rel="stylesheet" href="../css/index.css">
</head>

<body>
    <div class="container-fuild">
        <div class="banner">
            <div class="bg"></div>
            <div class="content">
                Youtube Lun Dev!
            </div>
        </div>
        <div class="asd" style="height: 100vh; width: 100px;"></div>
    </div>

</body>



</html>